app/src/main/java/net/squanchy/navigation/Navigator.java

```diff
@@ -38,6 +39,15 @@ public void toSearch() {
         context.startActivity(new Intent(context, SearchActivity.class));
     }
 
+    public void toContest() {
+        toContest(null);
+    }
+
+    public void toContest(String achievementId) {
```

Comment:
Same here, if you have two ways to open up the contest UI (with or without `achievementId`) see the comment on `ContestActivity`